mindmap/static/js/src/mindmapEdit.js

```diff
@@ -1,22 +1,87 @@
 /* Javascript for MindMapXBlock. */
 
 // TODO: add notifications
-function MindMapXBlock(runtime, element) {
+function MindMapXBlock(runtime, element, context) {
```

Comment:
why changing this too?

Reply:
I need the context similar to what `mindmap.js` does, it's to being able to render the mindmap in the edition panel